src/main/java/org/fcrepo/transform/http/FedoraTransform.java

```diff
@@ -108,26 +107,23 @@ public FedoraTransform(final String externalPath) {
      * @throws SecurityException if security exception occurred
      */
     @PostConstruct
-    public void setUpRepositoryConfiguration() throws RepositoryException, IOException {
+    public void setUpRepositoryConfiguration() throws RepositoryException, IOException, InvalidChecksumException {
 
-        final JcrTools jcrTools = new JcrTools(true);
         final Session internalSession = sessions.getInternalSession();
         try {
-            // register our CND
-            jcrTools.registerNodeTypes(internalSession, "ldpath.cnd");
-
-            // create the configuration base path
-            jcrTools.findOrCreateNode(internalSession, "/fedora:system/fedora:transform", "fedora:Configuration",
-                    "fedora:NodeTypeConfiguration");
-            final Node node =
-                jcrTools.findOrCreateNode(internalSession, CONFIGURATION_FOLDER + "default", NT_FOLDER, NT_FOLDER);
-            LOGGER.debug("Transforming node: {}", node.getPath());
-
-            // register an initial default program
-            if (!node.hasNode(NT_BASE)) {
-                final Node baseConfig = node.addNode(NT_BASE, NT_FILE);
-                jcrTools.uploadFile(internalSession, baseConfig.getPath(), getClass().getResourceAsStream(
-                        "/ldpath/default/nt_base_ldpath_program.txt"));
+
+            // containerService.findOrCreate(internalSession, "/fedora:system/fedora:transform");
+
+            final FedoraResource resource =
+                    containerService.findOrCreate(internalSession, CONFIGURATION_FOLDER + "default");
+            LOGGER.debug("Transforming resource: {}", resource.getPath());
+
+            final Stream<FedoraResource> children = resource.getChildren();
+            children.forEach(child -> LOGGER.debug("Child is {}", child.getPath()));
```

Comment:
Doh! This started to melt my mind. Yes, I will remove the extraneous logging. It is not useful and was just for my own sanity. Thanks to both of you.